cumulusci/cumulusci.yml

```diff
@@ -1,7 +1,14 @@
 cumulusci:
     keychain: cumulusci.core.keychain.EncryptedFileProjectKeychain
-
+    
 tasks:
+    activate_process_builder:
+        group: Salesforce Bulk API
```

Comment:
This is probably not the right group. Let's see if there's an existing group that fits, but I wonder if we need to start a new "Metadata Transformations" group.